storage/database/sharded_database.go

```diff
@@ -52,7 +52,7 @@ type sdbBatchResult struct {
 // newShardedDB creates database with numShards shards, or partitions.
 // The type of database is specified DBConfig.DBType.
 func newShardedDB(dbc *DBConfig, et DBEntryType, numShards uint) (*shardedDB, error) {
-	const numShardsLimit = 16
+	const numShardsLimit = 256
```

Comment:
How about moving this variable outside the function to package level constant?

Reply:
reflected.